plugins/@grouparoo/csv/src/lib/shared/parseProfiles.ts

```diff
@@ -1,30 +1,24 @@
 import fs from "fs-extra";
 import csvParser from "csv-parser";
-import {
-  plugin,
-  Source,
-  SourceMapping,
-  Run,
-  HighWaterMark,
-} from "@grouparoo/core";
+import { plugin, Source, SourceMapping, Run } from "@grouparoo/core";
 
 export async function parseProfiles({
```

Comment:
should this (and the file) be called `getRows` to match the other plugins?
not a big deal, but since we're doing so much renaming... 😛